hearth/sig/lib/hearth/client.rbs

```diff
@@ -0,0 +1,9 @@
+module Hearth
+  class Client
+    def initialize: (Hash[Symbol, untyped], Class) -> void
+
+    attr_reader config: untyped
```

Comment:
I think this would be parameterized right? Same for PluginList below?

Reply:
I couldn't figure out how to correctly parameterize it on the Class level.  Open to ideas.... The actual generated client classes all end up with correct typing.

Reply:
If they have the correct typing this this, then I think that's fine as is.